src/index.js

```diff
@@ -24,14 +32,12 @@ const pixabayjs = {
   },
 
   imageResultList: function(search, opts, onSuccess, onFailure) {
-    const url = 'https://pixabay.com/api';
-    const conf = this._makeConfig(url, opts);
+    const conf = this._makeConfig('', opts);
```

Comment:
Options come from the consumer, and it would defeat the purpose of having `imageResultList` and `videoResultList`